packages/altive_lints/lib/src/utils/types_utils.dart

```diff
@@ -82,8 +82,7 @@ bool _isWidget(DartType? type) => type?.getDisplayString() == 'Widget';
 bool _isSubclassOfWidget(DartType? type) =>
     type is InterfaceType && type.allSupertypes.any(_isWidget);
 
-// ignore: deprecated_member_use
-bool _isWidgetState(DartType? type) => type?.element2?.displayName == 'State';
```

Comment:
I was told to use `element` instead of `element2`, so I did!

Reply:
I agree!